homeassistant/components/vivotek/camera.py

```diff
@@ -33,12 +36,14 @@
         vol.Optional(CONF_SSL, default=False): cv.boolean,
         vol.Optional(CONF_VERIFY_SSL, default=True): cv.boolean,
         vol.Optional(CONF_FRAMERATE, default=2): cv.positive_int,
+        vol.Optional(CONF_STREAM_PATH, default=DEFAULT_STREAM_SOURCE): cv.string,
```

Comment:
Is it impossible to detect this automatically?

Reply:
This allows choosing a different stream path mostly to allow using a different encoding. I don't see a clear standard for this at the moment so that's why it's open-ended here.

Reply:
Ok.